src/main/java/seedu/address/logic/commands/LimitCommand.java

```diff
@@ -29,8 +34,22 @@ public LimitCommand(Threshold threshold) {
         this.threshold = threshold;
     }
 
+    /**
+     * @param hasNoArgument Limit for the Merit Score during borrowing.
+     */
+    public LimitCommand(boolean hasNoArgument) {
+        assert(hasNoArgument);
+
+        this.threshold = new Threshold();
+        this.hasNoArgument = hasNoArgument;
```

Comment:
I am not very sure, but won't this.hasNoArgument will always be True? If hasNoArgument == false, it will be forbit by `assert`.